rest/api.go

```diff
@@ -583,32 +587,38 @@ func (h *handler) handleHeapProfiling() error {
 
 func (h *handler) handlePprofGoroutine() error {
 	httpprof.Handler("goroutine").ServeHTTP(h.response, h.rq)
+	base.Audit(h.ctx(), base.AuditIDSyncGatewayProfiling, base.AuditFields{"profile_type": "goroutine"})
```

Comment:
is it worth marking `profile_type` a constant and marking the profile types as constants, since they are duplicated between our handlers that write to files and the `net/http/pprof` handlers.

Reply:
Agree about `profile_type`, left the `net/http/pprof` profile descriptions as-is.